src/main/java/me/hsgamer/bettereconomy/command/sub/TakeSubCommand.java

```diff
@@ -7,6 +7,8 @@
 
 import java.util.Optional;
 
+import me.hsgamer.bettereconomy.config.MessageConfig;
+
```

Comment:
Not needed

Reply:
To be fair, i had to use a tutorial to even know how a pull request works. I don't use GitHub